content/platforms/kubernetes/reference/redis_enterprise_cluster_api.md

```diff
@@ -0,0 +1,180 @@
+---
+Title: Redis Enterprise Cluster API
+description: 
+weight: 70
+alwaysopen: false
+categories: ["Platforms"]
+aliases:
+---
+
+This document describes the parameters for the Redis Enterprise Cluster custom resource.
+
+> Note this document is auto-generated from code comments. To contribute a change please change the code comments.
```

Comment:
```suggestion
> Note this document is auto-generated from the Operator code.
```
This isn't actionable for the docs audience since the repo where you can contribute these changes is closed source. Any contributions here will have to be addressed in that repo.